src/collections/posts/hidden-features-of-beatsaver.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Uncovering Hidden Features of BeatSaver
+section: articles
+category: guide
+publish: 2024-05-25T00:00:00.000Z
+homepageText: Not all BeatSaver features are apparent at first sight. This
+  article reveals hidden aspects of the BeatSaver platform such as tag
+  filtering, multi-tag selection, multi-adding to bookmarks, direct map search
+  via keys, double upload prevention, review curations, and displaying bot maps.
+image: /static/uploads/posts/uncovering-beatsaber-features-cover.png
+authors:
+  - '139574'
+showInPostListing: true
+linkToSpecialtyPage: /getting-started
+---
+
+[BeatSaver](https://beatsaver.com) is the place to get and distribute maps. If you are using a custom song mod it is more than likely that it retrieves from there. Some of BeatSaver's features can be a bit hidden or undocumented or only become apparent in specific cases.
+
+The following features are from the view of "normal users" and will thus go into aspects that are relevant when you are simply using the website to view and find maps or upload your own maps.
+
+# Excluding Tags
+
+By opening the filter dropdown you gain the ability to filter for tags. For example, if you are into Tech maps, you can search for Tech maps specifically.
+
+![](/uploads/posts/uncovering-beatsaber-features-1.png)
+
+At first glance, it appears to only allow for positive filtering; meaning you can only declare "I want to only be shown maps that have the XYZ tag".
+
+But what if you are having a lazy day and want to avoid Fitbeat maps? Just press the `[Alt]` button while clicking the tag that you want to exclude. You will find that the tag turns red, and maps with such a tag will no longer appear in your search!
+
+![](/uploads/posts/uncovering-beatsaber-features-2.png)
+
+# Selecting Multiple Tags
+
+When playing with the tag filters, you will notice that you can only select one of the blue tags and one of the green tags. Selecting another of the same category will result in the previous selection being deleted. However, it is possible to select multiple tags by pressing `[Shift]` while clicking the additional tags.
+
+![](/uploads/posts/uncovering-beatsaber-features-3.png)
+
+The results from the image above will now show you maps that either have the "Fitness" or "Tech" or both tags. If you were trying to find maps that are techy Fitbeat maps you cannot find them this way though.
+
+However, you can combine it with the tag exclusion feature and find your techy Fitbeat maps by excluding every tag that is not "Tech" and "Fitness" by pressing `[Shift]` and `[Alt]` together while clicking the tags.
+
+![](/uploads/posts/uncovering-beatsaber-features-4.png)
+
+This works because multiple negative filters will - other than the positive filters - result in being interpreted as "Neither X, Y, nor Z or any combination of those".
```

Comment:
Lines 39-45 are a bit confusing and possibly misleading unless I'm not following it correctly. Pressing Shift and Alt together will filter out all others, but also allows for maps that have no tags to display so you won't find your techy fitbeat maps. You can Shift select Tech and Fitbeat but it'll still be "either/or" instead of "and". As far as I know, there is no way to do the "and" query (I think this would've been part of the PR we previously discussed for BeatSaver).